joy2twist/config/joy2twist.yaml

```diff
@@ -0,0 +1,2 @@
+/joy2twist_node:
+  velocity_factor: { fast: 0.8, regular: 0.4, slow: 0.2 }
```

Comment:
Why it is not in a YAML format as it is in ros2 branch?

Reply:
Because in ROS1 we are able to easily read this velocity_factor parameter as a std::map (not supported in ROS2)

Reply:
Anyway I made changes here in order to obtain same way formatted yaml between ROS 1 and 2